.env

```diff
@@ -1,4 +1,5 @@
 REACT_APP_MURDOCK_PR_API_URL=https://ci.riot-os.org/api/pull_requests
 REACT_APP_MURDOCK_WS_URL=wss://ci.riot-os.org/status
 REACT_APP_NIGHTLIES_ROOT_URL=https://ci.riot-os.org
+REACT_APP_GITHUB_REPO=RIOT-OS/RIOT
```

Comment:
could you read this out of a subdomain? e.g., `https://<prefix-url>/RIOT-OS/RIOT` -> `RIOT-OS/RIOT`. With some backend work, this would allow more than one repo to be handled seamlessly.

Reply:
That would force the subpath to match the repository.

Reply:
When deploying, one can still set the env variable to a different repo when building the application:

```
REACT_APP_GITHUB_REPO=other/repo npm run build
```

And put the build result in the corresponding subpath.